### PR TITLE
Allow for markers and URIs being single values

### PR DESCRIPTION
**Pull request by MattyRoy**
I have investigated why I do not get ink/toner levels in Home Assistant for the IPP integration with my Canon MF230 laser printer (black only) when I stumbled upon [this](https://github.com/home-assistant/core/issues/87395) following open issue for the HA IPP integration.

After some digging I found the parsing to only cater for markers and URIs being list instances; I thus the changes in this pull request. Subsequently I see that there is already an open [#410 ](https://github.com/ctalkington/python-ipp/issues/410), which this PR should fix.

Debug log from Home Assistant showing the IPP data: [pyipp.log](https://github.com/ctalkington/python-ipp/files/11547824/pyipp.log)

### CODE REVIEW

File: src/pyipp/models.py [hunk 1]
```diff
@@ -178,30 +178,50 @@ def merge_marker_data(data):
...
                 if index < mlen:
                     marker_highs[index] = list_value
+        elif isinstance(data.get("marker-high-levels"), int) and mlen == 1:
+            marker_highs[0] = data.get("marker-high-levels")
 
         if isinstance(data.get("marker-low-levels"), list):
             for index, list_value in enumerate(data["marker-low-levels"]):
                 if index < mlen:
                     marker_lows[index] = list_value
+        elif isinstance(data.get("marker--low-levels"), int) and mlen == 1:
```
Comment (ctalkington) [inline src/pyipp/models.py:216]:
```suggestion
        elif isinstance(data.get("marker-low-levels"), int) and mlen == 1:
```

I think this dash is extra. Do you happen to have a debug dump from your printer? so I can put a proper test in place and make that's only issue.

Comment (ctalkington) [reply src/pyipp/models.py:216]: I may be able to use the log you provided to male a test. Looking into it now.

Comment (MattyRoy) [reply src/pyipp/models.py]: Thanks for that. Herewith the debug dump from my printer: 
[printer-attributes.zip](https://github.com/ctalkington/python-ipp/files/11562121/printer-attributes.zip)

**Comment by ctalkington** [comment]
Thanks!

**Pull request merged**